### ISSUE TITLE
Support Laravel 5.8

### ISSUE THREAD

**Issue by ThomHurks** [DarkaOnLine/L5-Swagger#196]
- L5-Swagger Version: 5.7.2
- PHP Version (`php -v`): 7.2
- OS: Ubuntu

### Description:
Laravel 5.8 has been released, and L5-Swagger is currently blocking our upgrade to it.

### Steps To Reproduce:
- Require Laravel 5.8
- ```composer update```

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: remove darkaonline/l5-swagger 5.7.2
    - Conclusion: don't install darkaonline/l5-swagger 5.7.2
    - Conclusion: don't install darkaonline/l5-swagger 5.7.1
    - Can only install one of: laravel/framework[5.8.x-dev, 5.6.x-dev].
    - Can only install one of: laravel/framework[v5.8.0, 5.6.x-dev].
    - darkaonline/l5-swagger 5.7.0 requires laravel/framework 5.6.* -> satisfiable by laravel/framework[5.6.x-dev].
    - Installation request for darkaonline/l5-swagger 5.7.* -> satisfiable by darkaonline/l5-swagger[5.7.0, 5.7.1, 5.7.2].
    - Installation request for laravel/framework 5.8.* -> satisfiable by laravel/framework[5.8.x-dev, v5.8.0].
```

**Comment by DarkaOnLine** [comment]
Released 5.8.0 -  https://github.com/DarkaOnLine/L5-Swagger/releases/tag/5.8.0

**Comment by ThomHurks** [comment]
Thanks for the fast response, @DarkaOnLine!
